### ISSUE TITLE
cli export data fail “no querier node online”

### ISSUE THREAD

**Issue by code-jar** [openobserve/openobserve#5078]
### Which OpenObserve functionalities are the source of the bug?

Don't known / other

### Is this a regression?

No

### Description

`openobserve.exe export --stream_type traces --start 1730304000 --end 1732032000 --data . /202411` 
![fail](https://github.com/user-attachments/assets/b2d35215-92ac-4b69-8ad6-e652819e1e6e)
But the export log is executed successfully, because the log is empty.
![success](https://github.com/user-attachments/assets/f8128d89-f7cf-4f45-b209-db945ba0acd4)

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version: v0.13.0
OS: win10-x64
```

**Comment by code-jar** [comment]
> @code-jar could you share your env config. Mainly `ZO_LOCAL_NODE`您能否分享您的 env 配置。主要`ZO_LOCAL_NODE`

I only set the default
![image](https://github.com/user-attachments/assets/caf04073-7059-47b8-95d6-d4e86bcdcfdf)

**Comment by Loaki07** [comment]
will check this and fix it

**Comment by code-jar** [comment]
> @code-jar could you share your env config. Mainly `ZO_LOCAL_NODE`

Did you mean 'ZO_LOCAL_MODE'?   NO,  I only set the default
![image](https://github.com/user-attachments/assets/caf04073-7059-47b8-95d6-d4e86bcdcfdf)

**Comment by code-jar** [comment]
Any progress?

**Comment by hengfeiyang** [comment]
I think @Loaki07 have no time on this case, still need wait for a few days.

**Comment by hengfeiyang** [comment]
@code-jar i think @Loaki07 already start working on this case

**Comment by code-jar** [comment]
😞the version keeps getting updated, but my problem has never been solved, which makes me sad!

**Comment by hengfeiyang** [comment]
@code-jar first is not easy to fix. second, this feature already exist over 1 year, you are the first to use it, actually, i am thinking remove it. even we fix it, maybe it not work like you expect.

So, before we remove it, i will fix it later. but maybe later will remove it.

**Comment by hengfeiyang** [comment]
Can you give you use case, you want to use the export feature to do what? how many data you want to export. and what is the format you are expect?

**Comment by code-jar** [comment]
> Can you give you use case, you want to use the export feature to do what? how many data you want to export. and what is the format you are expect?

Currently I'm using sqlserver to record tracking information and periodically archive production environment data locally, one to minimize production environment data Also locally I can trace historical data. I would like to replace it with openobserve

**Comment by code-jar** [comment]
> @code-jar first is not easy to fix. second, this feature already exist over 1 year, you are the first to use it, actually, i am thinking remove it. even we fix it, maybe it not work like you expect.
> 
> So, before we remove it, i will fix it later. but maybe later will remove it.

okay 😞

**Comment by hengfeiyang** [comment]
@code-jar how many data you want to export. and what is the format you are expect? we can discuss your case.

**Comment by code-jar** [comment]
> @code-jar how many data you want to export. and what is the format you are expect? we can discuss your case.

I don't care about the format of the data, I just want to be able to export from one instance and import to another. Preferably export by time period

**Comment by hengfeiyang** [comment]
the question is, if your data is 1GB, it is should be okay, but if your data is 100GB or 1TB the exporter maybe can't work.

**Comment by hengfeiyang** [comment]
@code-jar, i got an idea let it work:
1. start the OpenObserve service
2. run the import / export again

it will work, try again. recently we add some command depends on the service need run first, i think this issue always can resolve like this.

**Comment by code-jar** [comment]
> [@code-jar](https://github.com/code-jar), i got an idea let it work:
> 
> 1. start the OpenObserve service
> 2. run the import / export again
> 
> it will work, try again. recently we add some command depends on the service need run first, i think this issue always can resolve like this.

需要升级最新预览版嘛？当前是v0.14.3

![Image](https://github.com/user-attachments/assets/24b68f4c-7590-4af4-b734-853d4237bd0d)

**Comment by hengfeiyang** [comment]
you need run the service first, you can access it thought http, then run the command

**Comment by code-jar** [comment]
> you need run the service first, you can access it thought http, then run the command

我确认我的服务已经运行起来了，跟运行的环境有关吗？目前是 windows下通过nssm部署的服务。然后到程序的文件夹执行的导出命令。
当我使用默认命令 `openobserve.exe export --data ./backup`执行无报错，但是使用 `openobserve.exe export --data ./haha --stream_type traces` 就会提示 "no querier node online"

**Comment by hengfeiyang** [comment]
Thank you for report, let me think about it more.

**Comment by hengfeiyang** [comment]
You can try this version:

public.ecr.aws/zinclabs/openobserve-dev:v0.14.5-rc1-7fded1f

**Comment by hengfeiyang** [comment]
You can try `v0.14.5-rc2` we just released the new version.

**Comment by code-jar** [comment]
> You can try `v0.14.5-rc2` we just released the new version.

👍   但是我发现好像限制了条数，最大只有100条数据

**Comment by hengfeiyang** [comment]
Yes, this feature looks never working. maybe you can leave it. i will try to optimize it later.

**Comment by hengfeiyang** [comment]
That issue already improved but need wait for next release, then you can use it like this:
```
ZO_QUERY_DEFAULT_LIMIT=10000 ./openobserve export --
